src/app/work-item/work-item-quick-add/work-item-quick-add.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="quickAdd_container" id="workItemQuickAdd_container" >
+<div class="quickAdd_container" id="workItemQuickAdd_container" *ngIf="wilistview === 'wi-list-view'" >
```

Comment:
We have camel case pattern at all places. If we dont want camel case in IDs and Classes it should be a separate issue